geode-core/src/integrationTest/java/org/apache/geode/internal/cache/wan/asyncqueue/AsyncEventQueueValidationsJUnitTest.java

```diff
@@ -71,6 +76,39 @@ public void testConcurrentParallelAsyncEventQueueAttributesWrongDispatcherThread
     }
   }
 
+  @Test
+  @Parameters({"true", "false"})
+  public void whenAEQCreatedInPausedStateThenSenderIsStartedInPausedState(boolean isParallel) {
+    cache = new CacheFactory().set(MCAST_PORT, "0").create();
+    AsyncEventQueueFactory fact = cache.createAsyncEventQueueFactory()
+        .setParallel(isParallel)
+        .pauseEventDispatchingToListener()
+        .setDispatcherThreads(5);
+    AsyncEventQueue aeq =
+        fact.create("aeqID", new org.apache.geode.internal.cache.wan.MyAsyncEventListener());
+    assertTrue(((AsyncEventQueueImpl) aeq).getSender().isPaused());
+  }
+
+  @Test
+  @Parameters({"true", "false"})
+  public void whenAEQCreatedInPausedStateIsUnPausedThenSenderIsResumed(boolean isParallel) {
+    cache = new CacheFactory().set(MCAST_PORT, "0").create();
+    AsyncEventQueueFactory fact = cache.createAsyncEventQueueFactory()
+        .setParallel(isParallel)
+        .pauseEventDispatchingToListener()
+        .setDispatcherThreads(5);
+    AsyncEventQueue aeq =
+        fact.create("aeqID", new org.apache.geode.internal.cache.wan.MyAsyncEventListener());
+    assertTrue(aeq.isDispatchingPaused());
```

Comment:
Wouldn't this line better fit in the test above? and then you can remove these two asserts this test (since they would be covered by the test above).

Reply:
Done